conans/client/toolchain/msbuild.py

```diff
@@ -0,0 +1,99 @@
+import os
+import textwrap
+from xml.dom import minidom
+
+from conans.client.tools import msvs_toolset
+from conans.util.files import save, load
+
+
+class MSBuildToolchain(object):
+
+    def __init__(self, conanfile):
+        self._conanfile = conanfile
+        self.definitions = {}
+
+    @ staticmethod
+    def _name_condition(settings):
+        toolset = msvs_toolset(settings)
+
+        props = [("Configuration", settings.build_type),
+                 # FIXME: This probably requires mapping ARM architectures
+                 ("Platform", {'x86': 'Win32',
```

Comment:
yes, `armv7` -> `ARM`, `armv8` - `ARM64` (maybe few others for Windows CE and Itanium).
https://github.com/dotnet/msbuild/blob/df645299a158c787f9884fca2bcd8c295318193f/src/Shared/NativeMethodsShared.cs#L437
https://github.com/dotnet/msbuild/blob/cb5277fbfa4fba3e87874b472ade356c0ecf47b2/src/Utilities/ProcessorArchitecture.cs#L60
might be good idea to extract to the common tool as well, as we have mapping in several places already